src/airflow/dags/test_DAG.py

```diff
@@ -0,0 +1,170 @@
+        "source_object": "credible_set_datasets/gwas_catalog_curated",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/credible_set/gwas_catalog_curated",
+    },
+    # PICS credible sets from GWAS Catalog summary statistics:
+    "gwas_catalog_sumstats_credible_set": {
+        "source_bucket": GWAS_CATALOG_BUCKET_NAME,
+        "source_object": "credible_set_datasets/gwas_catalog_summary_stats",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/credible_set/gwas_catalog_summary_statistics",
+    },
+    # GWAS Catalog manifest files:
+    "gwas_catalog_manifests": {
+        "source_bucket": GWAS_CATALOG_BUCKET_NAME,
+        "source_object": "manifests",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/manifests",
+    },
+    # eQTL Catalog study index:
+    "eqtl_catalog_study_index": {
+        "source_bucket": EQTL_BUCKET_NAME,
+        "source_object": "study_index",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/study_index/eqtl_catalog",
+    },
+    # eQTL Catalog SuSiE credible sets:
+    "eqtl_catalog_susie_credible_set": {
+        "source_bucket": EQTL_BUCKET_NAME,
+        "source_object": "credible_set_datasets/susie",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/credible_set/eqtl_catalog_susie",
+    },
+    # Finngen study index:
+    "finngen_study_index": {
+        "source_bucket": FINNGEN_BUCKET_NAME,
+        "source_object": f"{FINNGEN_RELEASE}/study_index",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/study_index/finngen",
+    },
+    # Finngen summary statistics:
+    "finngen_PICS_credible_set": {
+        "source_bucket": FINNGEN_BUCKET_NAME,
+        "source_object": f"{FINNGEN_RELEASE}/credible_set_datasets/finngen_pics",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/credible_set/finngen_pics",
+    },
+    # Finngen SuSiE credible sets:
+    "finngen_susie_credible_set": {
+        "source_bucket": FINNGEN_BUCKET_NAME,
+        "source_object": f"{FINNGEN_RELEASE}/credible_set_datasets/finngen_susie_processed",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/credible_set/finngen_susie",
+    },
+    # L2G gold standard:
+    "gold_standard": {
+        "source_bucket": "genetics_etl_python_playground",
+        "source_object": "input/l2g/gold_standard/curation.json",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/locus_to_gene_gold_standard.json",
+    },
+}
+
+
+# Test if release folder exists:
+def test_release_folder_exists() -> bool:
```

Comment:
This util might be useful as a sensor in other instances.
You could parametrise it and move it to `common_airflow.py`

```python
def test_release_folder_exists(bucket: str, path: str) -> bool:
    """This function tests if the release folder exists.

    Args:
        bucket (str): Name of the bucket in GCS
        path (str): Object name in GCS to check 

    Returns:
        bool: False if the folder exists, True otherwise.
    """
    hook = GCSHook(gcp_conn_id="google_cloud_default")
    return not hook.exists(bucket, path)
```
Minor, but the GCSHook connector is interesting

Reply:
Agree, it makes sense to generalise.